.github/workflows/ci.yml

```diff
@@ -148,7 +148,7 @@ jobs:
         with:
           repository: 'oceanprotocol/ocean-node'
           path: 'ocean-node'
-          ref: 'fix/transferencoding+contentleght'
+          ref: 'main'
```

Comment:
It's not needed to specify the main branch of a repo, only delete the existing branch and we are ready for merging.
Thanks!

Reply:
true 👍